install/create_system_config_cds.md

```diff
@@ -0,0 +1,394 @@
+     ```
+
+     > **NOTE**: The `iprange` for single VLAN or `mtliprange`, `nmniprange`, and `hmniprange` for HMN configured network are available in the `system_config.yaml` file in the `prep` directory.
+
+### 3.2 Verify and reset the BMCs
+
+The following steps will verify if IPs are assigned to the BMC nodes and reset the nodes:
+
+1. (`pit#`) If you want to dynamically discover the fabric switches, PDUs, and CMCs, ensure they are set to `dhcp` mode.
+
+1. (`pit#`) List the BMC nodes.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_start.py -u bmc_username -p bmc_password --list
+   ```
+
+1. Verify that all the required nodes are listed.
+
+   Example output:
+
+   ![List the BMC nodes](../img/install/List_BMC_nodes.png)
+
+1. (`pit#`) Reset the listed BMC nodes.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_start.py -u bmc_username -p bmc_password --reset
+   ```
+
+### 3.3 Verify the discover status
+
+> **NOTE**: If the fabric switches, PDUs, and CMCs were not configured to `dhcp` mode in the step [Verify and reset the BMCs](#32-verify-and-reset-the-bmcs), manually create the `fabricswlist`, `pdulist`, and `cmclist` files in the working directory.
+
+The following steps verify the status and lists the IPs of nodes, fabric switches, PDUs, and CMCs:
+
+1. (`pit#`) Verify the status and list the IPs of the nodes.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py nodes --out
+   ```
+
+   1. (`pit#`) Create a `bond0` network interface on all the listed nodes.
+
+      ```bash
+      pdsh -w^nodelist -x localhost "sh "${CVT_PATH}"/scripts/create_bond0.sh"
+      ```
+
+   1. Ensure that `bond0` network interface is created on all the nodes listed in the `nodelist`.
+
+      Example output:
+
+      ![Bond0 Network Interface](../img/install/create_bond0.png)
+
+1. (`pit#`) Verify the status and list the IPs of the fabric switches.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py fabric --username fabric_username --password fabric_password --out
+   ```
+
+1. (`pit#`) Verify the status and list the IPs of the PDUs.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py pdu --username pdu_username --password pdu_password --out
+   ```
+
+1. (`pit#`) Verify the status and list the IPs of the CMCs.
+
+   ```bash
+   "${CVT_PATH}"/cds/discover_status.py cmc --username cmc_username --password cmc_password --out
+   ```
+
+### 3.4 Collect and compare the inventory data
+
+#### 3.4.1 Dynamic Data Collection
+
+1. (`pit#`) Copy the SSH key to setup passwordless-SSH for the localhost.
+
+   ```bash
+   ssh-copy-id localhost
+   ```
+
+1. (`pit#`) Collect the hardware inventory data.
+
+   ```bash
+   "${CVT_PATH}"/system_inventory.py --nodefile nodelist
+   ```
+
+1. (`pit#`) Collect the fabric inventory data.
+
+   ```bash
+   "${CVT_PATH}"/fabric_inventory.py --switchfile ./cds/fabricswlist --username switch_username --password switch_password
```

Comment:
The user should not specify the password on the command line. See the previous comment about this earlier in the page.